src/functions-and-arrays.js

```diff
@@ -78,7 +143,15 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(arrayOrtiz) {
+  let nothing = 0;
+  for (let i = 0; i < arrayOrtiz.length; i++) {
+    if (arrayOrtiz[i] === wordsCount) {
+      nothing++;
+    }
+  }
+  return nothing;
```

Comment:
For this iteration your function should receive two parameters: the word that you want to count, and the array where you want to search for the word.
Your code is not working because you only have one parameter in your function, and you're calling the name of the array inside the function (wordsCount) instead of calling the placeholder (wordToCount)

```
function howManyTimes(arrayToSearch, wordToCount) {
  let nothing = 0;
  for (let i = 0; i < arrayToSearch.length; i++) {
    if (arrayToSearch[i] === wordToCount) {
      nothing++;
    }
  }
  return nothing;
}
```